htmlToElement.js

```diff
@@ -16,24 +17,39 @@ const defaultOpts = {
 };
 
 const Img = props => {
-  const width =
+  let widthAtt =
     parseInt(props.attribs['width'], 10) || parseInt(props.attribs['data-width'], 10) || 0;
   const height =
     parseInt(props.attribs['height'], 10) ||
     parseInt(props.attribs['data-height'], 10) ||
     0;
 
+  const { width } = Dimensions.get('window');
+  const resizeMode = null;
+
+  if (!widthAtt) {
+    widthAtt = (width - 20);
+    resizeMode: 'contain'
```

Comment:
I feel like some of this stuff would be better off in `AutoSizedImage`, which after all is supposed to take care of its own sizing :smile: